### PR TITLE
Adds July 26th agenda

### PR DESCRIPTION
**Pull request by calvincestari**
Adding the agenda for our next meeting.

### CODE REVIEW

**Comment by calvincestari** [comment]
@benjie, please double-check the meeting link shows the correct time in your timezone.

**Comment by calvincestari** [comment]
I don't think invitations are going out for this meeting so I'm pinging a list of people in this comment that may be interested in attending:
- @twof 
- @fotoetienne 
- @aprilrd 
- @mjmahone 
- @AnthonyMDev
- @BobaFetters
- @martinbonnin
- @BoD

Please let others know that you think may be interested - thanks.

**Comment by aprilrd** [comment]
Yup, I will attend. Should I create a separate PR to add myself to the list?

**Comment by calvincestari** [comment]
> Yup, I will attend. Should I create a separate PR to add myself to the list?

@aprilrd, you can probably just make a suggestion comment on line 100 of this PR and I'll accept the suggestion.

File: agendas/2023/2023-07-26.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  https://calendar.google.com/calendar?cid=bGludXhmb3VuZGF0aW9uLm9yZ19pazc5dDl1dWoycDMyaTNyMjAzZGd2NW1vOEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t
+[ical file]:
+  https://calendar.google.com/calendar/ical/linuxfoundation.org_ik79t9uuj2p32i3r203dgv5mo8%40group.calendar.google.com/public/basic.ics
+
+## Attendees
+
+<!--prettier-ignore -->
+| Name               | GitHub            | Organization            | Location                 |
+| :----------------- | :---------------- | :---------------------- | :----------------------- |
+| Calvin Cestari     | @calvincestari    | Apollo                  | Vancouver, BC, CA        |
```
Comment (fotoetienne) [inline agendas/2023/2023-07-26.md:100]:
```suggestion
| Calvin Cestari     | @calvincestari    | Apollo                  | Vancouver, BC, CA        |
| Stephen Spalding   | @fotoetienne      | Netflix                 | Los Gatos, CA, US       |
```

File: agendas/2023/2023-07-26.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+| Stephen Spalding   | @fotoetienne      | Netflix                 | Los Gatos, CA, US       |
```
Comment (aprilrd) [inline agendas/2023/2023-07-26.md:101]:
```suggestion
| Stephen Spalding   | @fotoetienne      | Netflix                 | Los Gatos, CA, US       |
| Young Min Kim      | @aprilrd          | The Trade Desk          | San Francisco, CA, US   |
```

File: agendas/2023/2023-07-26.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+This is an open meeting in which anyone in the GraphQL community may attend.
+
+- **Date & Time**:
+  [July 26th 2022 19:00 - 20:00 UTC](https://www.timeanddate.com/worldclock/meetingdetails.html?year=2023&month=7&day=26&hour=19&min=0&sec=0&p1=224&p2=179&p3=136&p4=268&p5=367&p6=438&p7=240&iv=0)
+  - View the [calendar][], or subscribe ([Google Calendar][], [ical file][]).
+  - _Please Note:_ The date or time may change. Please check this agenda the
+    week of the meeting to confirm. While we try to keep all calendars accurate,
+    this agenda document is the source of truth.
+- **Video Conference Link**: https://zoom.us/j/95723407712?pwd=ZHhQakxqd1JlTnZJNTJXdlN3UFdQUT09
+  - _Password:_ ccn-ftw
```
Comment (benjie) [inline agendas/2023/2023-07-26.md:84]:
```suggestion
  - _Password:_ ccnftw
```

**Pull request merged**